PagingSample/app/src/androidTest/java/paging/android/example/com/pagingsample/MainActivityTest.kt

```diff
@@ -19,30 +19,24 @@ import android.content.Intent
 import androidx.recyclerview.widget.RecyclerView
 import androidx.test.core.app.ActivityScenario
 import androidx.test.core.app.ApplicationProvider
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import org.hamcrest.CoreMatchers
-import org.hamcrest.MatcherAssert
+import com.google.common.truth.Truth.assertThat
 import org.junit.Test
-import org.junit.runner.RunWith
-import java.util.concurrent.TimeoutException
 
 /**
  * Simply sanity test to ensure that activity launches without any issues and shows some data.
  */
-@RunWith(AndroidJUnit4::class)
```

Comment:
why is this removed?

Reply:
Oops - done, was getting a test not found issue earlier so was testing this temporarily :|